src/main/kotlin/no/nav/medlemskap/services/pdl/PdlMapper.kt

```diff
@@ -1,95 +1,77 @@
 package no.nav.medlemskap.services.pdl
 
-import no.nav.medlemskap.common.exceptions.DetteSkalAldriSkje
 import no.nav.medlemskap.common.exceptions.PersonIkkeFunnet
 import no.nav.medlemskap.domene.*
 import no.nav.medlemskap.domene.Familierelasjon
 import no.nav.medlemskap.domene.Folkeregistermetadata
 import no.nav.medlemskap.domene.Sivilstand
 import no.nav.medlemskap.domene.Statsborgerskap
+import no.nav.medlemskap.services.pdl.PdlSivilstandMapper.mapSivilstander
 
-fun mapTilPersonHistorikk(person: HentPdlPersonResponse): Personhistorikk {
-    val statsborgerskap: List<Statsborgerskap> = person.data?.hentPerson?.statsborgerskap?.map {
-        Statsborgerskap(
-                landkode = it.land,
-                fom = it.gyldigFraOgMed,
-                tom = it.gyldigTilOgMed
-        )
-    } ?: throw PersonIkkeFunnet("PDL")
+object PdlMapper {
```

Comment:
Hva er fordelen med å ha det som et Object fremfor som en function?

Reply:
Ved å ha det definert i Object, kan man finne PdlMapper ved å søke etter klasse. Jeg synes dessuten det er en god ide å gruppere ting som har noe med hverandre å gjøre, og ikke ha det som frittstående funksjoner

Reply:
Ah, det er en god nok grunn for meg. Jeg klikker Cmd-O og begynner å skrive, og da er det veldig fint når det dukker opp noe! :)